framework/doc/content/syntax/MultiApps/index.md

```diff
@@ -45,28 +45,63 @@ equations can be solved with [Fixed Point iterations](syntax/Executioner/index.m
 
 !listing multiapps/transient_multiapp/dt_from_parent.i block=MultiApps
 
-## Positions id=multiapp-positions
+The input file(s) for the sub-app(s) are specified using the [!param](/MultiApps/TransientMultiApp/input_files)
+parameter. If only one input file is provided, then this input file is used for all
+sub-apps in this `MultiApp`.
+
+!alert! warning title=Multiple input files per application not possible with `MultiApp`
+The ability to specify multiple input files per application, e.g.,
 
-The [!param](/MultiApps/TransientMultiApp/positions) parameter is a coordinate offset from
-the main app domain to the sub-app domain, as illustrated by [multiapps_pos]. The parameter
-requires the positions to be provided as a set of $(x, y, z)$ coordinates for each sub-app.
+```
+subapp-opt -i input1.i input2.i
+```
 
-The number of coordinate sets determines the actual number of sub-applications created.  If there is
-a large number of positions a file can be provided instead using the
-[!param](/MultiApps/TransientMultiApp/positions_file) parameter.
+is not possible with `MultiApp`s, since each input file in [!param](/MultiApps/TransientMultiApp/input_files)
+is interpreted as a different application.
+!alert-end!
```

Comment:
I think it can be done through the cli_args but I dont remember

Reply:
I'll just remove the statement - I don't think it's necessary.

Reply:
On second thought, I think it needs some kind of discussion there - I know at least two people that have tried this.